### PR TITLE
Apply webpack configuration to our current state in source directory

### PR DESCRIPTION
**Pull request by xeho91**
Hi folks!

I had to create a new Pull Request from #18, as the last one was very conflicted, that even I got lost at some point.
I tried to resolve this mess, but without success, so hence why I decided to create a new one.
This one should be cleaner, and involve only two commits. This way, is easier to see what extactly has changed in the `./source` directory, to see the example usage with webpack.

### CODE REVIEW

**Comment by xeho91** [comment]
> 
> 
> That was a lot of changes. I am glad that you understand the config files. I think it is going to take me a minute to figure out what is happening here. lol

Just a minute? :smile:

**Comment by lax0248917** [comment]
I meant that in the same way as "I'm going to get my "hair" cut" by that of course I mean all my hair...so in fact I meant "All" the minutes! 😂

File: webpack.config.js [hunk 1]
```diff
@@ -54,6 +54,19 @@ var directories = new (function Directories() {
...
+	// https://github.com/jantimon/html-webpack-plugin#options
+	inject: "head",
+	meta: {
+		viewport: "width=device-width, initial-scale=1",
+	},
+};
+var pageTitles = {
+	"about-us": "About Us",
+	"contact-us": "Contact Us",
+	projects: "Projects",
```
Comment (mariushilleke) [inline webpack.config.js:67]: Was it intentionally that the `projects`-key does not have quotes?

Comment (mariushilleke) [reply webpack.config.js:67]: Because of the missing `-` dash in that key?

Comment (xeho91) [reply webpack.config.js:67]: If there's a hyphen in the name, then you need to wrap it withing quotes. At least inside JavaScript files.

File: webpack.config.js [hunk 2]
```diff
@@ -54,6 +54,19 @@ var directories = new (function Directories() {
 	this.templates = path.join(this.source, "templates");
 })();
 
+var sharedHTMLwebpackPluginOptions = {
```
[the remark this answers is not in the document]
Comment (xeho91) [reply webpack.config.js:57]: I personally prefer to have HTML uppercased, so I wasn't sure which one of these formats to pick. camelCase doesn't always look pretty. 😕 

- HTMLWebpackPlugin
- HTMLwebpackPlugin <- selected
- HtmlWebpackPlugin <- please no 😭 

Thoughts?

[the remark this answers is not in the document]
Comment (xeho91) [reply webpack.config.js:57]: Just for the reference:
https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms

[the remark this answers is not in the document]
Comment (mariushilleke) [reply webpack.config.js:57]: Hmmm... ok 😄 
that point in the reference got me:

> Why? Names are for readability, not to appease a computer algorithm.

**HTMLwebpackPlugin ** it is 🥳

**Pull request merged**